src/bare_etiss_processor/base.ini

```diff
@@ -22,8 +22,3 @@ ETISS::write_pc_trace_from_time_us=0
 ETISS::write_pc_trace_until_time_us=3000000
 ETISS::sim_mode=0
 vp::simulation_time_us=20000000
-
-[Plugin Logger]
```

Comment:
There is no way to "out-out" from using the Logger plugin, hence I would go for an "opt-in" appraoch, potentially breaking users scripts which run `bare_etiss_processor` directly (without `run_helper.sh`) and use the legacy syscalls. Any comments?